pyplanet/apps/core/pyplanet/templates/toolbar.xml

```diff
@@ -29,10 +29,15 @@
          id="player_toolbar_bar_button_jb"
          scriptevents="1" action="{{ id }}__bar_button_jb" />
 
-  <label pos="22.75 -3.2" z-index="1" size="6.5 6" text="🕑" halign="center" valign="center2" id="player_toolbar_bar_button_extend" textsize="2.5"
-         scriptevents="1" action="{{ id }}__bar_button_extend" bgcolor="0000" />
-  <label pos="23.75 -4.5" z-index="1" size="3 3" text="" halign="center" valign="center2"
-         textsize="0.5" opacity="1" focusareacolor1="0000" focusareacolor2="0000" />
+  {% if game == 'sm' %}
+    <label pos="22.75 -3.2" z-index="1" size="6.5 6" text="&#xf079;" halign="center" valign="center2" id="player_toolbar_bar_button_replay" textsize="2.5"
```

Comment:
Did you check if the icon works on SM? @TheMaximum

Reply:
I checked the icon working in TM with the game ManiaLink context data hardcoded as `sm`. Can that differ?